executor/cook/__main__.py

```diff
@@ -21,6 +21,11 @@
 
 
 def main(args=None):
+    if len(sys.argv) == 2 and sys.argv[1] == "--version":
+        from _version import __version__
+        print(__version__)
+        sys.exit(0)
+
     print('Starting cook executor...')
```

Comment:
Can we include the version here?

Reply:
How do you want it formatted?

Reply:
I like: `print('Starting cook executor {}...'.format(version))`